### DISCUSSION TITLE
Convert screen offset to chart coordinates

### DISCUSSION THREAD

**Discussion by KapitanPL** [imaNNeo/fl_chart#1135] [Q&A] [answered]
This should be simple, yet I struggle with it. 

How do I convert an offset in pixels to chart coordinates? (or the other way...)

**Comment by imaNNeo** [comment, marked as answer]
Unfortunately, there is no built-in way to do that. Did you open an issue?

**Reply by KapitanPL** [reply]
Nope. This does not seem as an real issue to me. Unless you see it as issue. 
But maybe a guide/idea ho to do it? 

I found a way how you do it, but it is rather convoluted and gathering pieces of information here and there. 
If you have an idea how it can be /relatively easily/ achieved I am willing to try to write it myself and do a pullrequest.

But if there does not seem to be an easy way (and I guess the issue might not be fixed fast) I guess I will consider using another chart widget. Would be pitty, this one is already pretty integrated.

**Reply by imaNNeo** [reply]
Yes, it can be an idea. Thanks for creating an issue.
